### ISSUE TITLE
Idea for using PS data to create categories of functionality of sidealks

### ISSUE THREAD

**Issue by yeisenberg** [ProjectSidewalk/SidewalkWebpage#3056]
##### Brief description of problem/feature
 Austin TX shared their great work on Sidewalks. I think they are a leader in this area as far as thinking/strategy. This is an approach they used to rank priority. I was thinking this could be great to see if we can visualize/adapt PS data into these categories as a way to make an option for a direct usable end product for communities to prioritize improvements.
![image](https://user-images.githubusercontent.com/13573781/197202169-b903d00d-f6af-4a7a-89c0-bc92d8f3c9b0.png)

We can try to categorize to help communities we are working with to prioritize

**Comment by misaugstad** [comment]
There are some difficulties with matching to a chart like this, because so much of it has to do with precise measurements of widths, slops, vertical clearance, etc. We would have to narrow it to the aspects that our data can actually speak to!

**Comment by yeisenberg** [comment]
Right! I think that is where I was going with it. What data is in PS that could be used to get at these categories.

--
Yochai Eisenberg, PhD
***@***.******@***.***>
312-413-9410

**Comment by misaugstad** [comment]
* Width: we have the 'narrow' tag. Though this is binary, and it is used in tandem with the Surface Problem and Obstacle label types, so it's not always about the width of the sidewalk, but about the _available_ width of the sidewalk due to obstrucitons.
* Cross-slope: Both the 'not level with street' and 'steep' tags for Curb Ramp labels sort of gets at this, but not in a super meaningful way.
* Faults -- not 100% sure what they mean by this, we can loop back to it!
* Cracks: we have the 'cracks' and 'very broken' tags for Surface Problem labels! Combining that with severity gets at this pretty well.
* Vertical clearancec: we don't have anything great for this... It would fall under the Obstacle label type, but we don't have any good tags for it
* Obstruction: We have Obstacle labels!

**Comment by yeisenberg** [comment]
Nice, thanks for that layout!

  *   Cross slope - could we teach people to use the surface problem and slanted tag?
  *   Vertical clearance - Would adding a tag help with that if we came up with a clear one?

I'll follow up on the Faults definition.

thanks



--
Yochai Eisenberg, PhD
***@***.******@***.***>
312-413-9410
